tests/test_bigquery_etl.py

```diff
@@ -0,0 +1,39 @@
+import os
+from dataclasses import dataclass
+from pathlib import Path
+import tempfile
+from unittest.mock import patch
+
+from sync.bigquery_etl import get_bigquery_etl_table_references
+import tarfile
+
+
+@dataclass
+class MockApiResponse:
+    content: bytes
+
+
+@patch("requests.get")
+def test_get_legacy_pings(mock_get):
+    with (tempfile.NamedTemporaryFile(suffix=".tar.gz") as fp,):
```

Comment:
Probs good to delete the temp file after the test finishes.
```suggestion
    with (tempfile.NamedTemporaryFile(suffix=".tar.gz") as fp, delete=True):
```

Reply:
Deleting is the default behaviour for `tempfile` context managers